.travis.yml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+---
+language: python
+python:
+  - 2.6
+  - 2.7
+  - pypy
+  - 3.2
+  - 3.3
+  - 3.4
```

Comment:
I wouldn't bother with all these versions, personally. We should target 3.4. If that's impossible, it should work with 2.7. If we can't do that, we need to jump up and down and wave arms, because I don't really want to see us in 2.6 land anymore.